overlay.d/50scos/usr/lib/okd/selinux/afterburn.cil

```diff
@@ -0,0 +1,7 @@
+(typeattributeset cil_gen_require var_run_t)
```

Comment:
will these rules also persist after the bootimage pivots to the scos image?

Reply:
yes

Reply:
I'd also say that we could delete this file once a proper fix lands in CS9? I didn't reproduce in a CS9 fresh env with no other modifications as the ones here, but it seems a bug to file there for afterburn or the selinux-policy package you mentioned in the issue.

Reply:
once we have scos publicly available, yes we would need this. But by that time I hope that we can get the actual issue with selinux-policy sorted out.

Reply:
https://github.com/fedora-selinux/selinux-policy/pull/1362/files might be the fix to downstream?

Reply:
And https://github.com/fedora-selinux/selinux-policy/pull/2000/files

Reply:
@travier see the two PRs above for https://github.com/openshift/os/pull/1556#issuecomment-2239291679